app/controllers/api/v1/books_controller.rb

```diff
@@ -2,38 +2,29 @@ module Api
   module V1
     class BooksController < ApplicationController
       def index
-        books = Book.all
+        raise StandardError, "Unable to serialize books" if serialized_books.nil?
```

Comment:
You've gone down the wrong path here.

But it's very understandable why you have, as we went down that path in Exceptions 1.

Here, you shouldn't be making any changes to the actual code. Only the test code. 

We'll use RSpec's mocking behaviour to mock an exception being raised.

Not actually raise an exception in the real code.

We can book in some extra time to pair on this if you like? Let me know.

Reply:
That would be great, thanks! Can you still meet tomorrow or shall we meet up on 28/7?

Reply:
I've sent more detail in an email but, yes the 28th sounds good :)